addons/website/static/src/components/navbar/navbar.scss

```diff
@@ -0,0 +1,8 @@
+.o_menu_systray.o_website_systray {
+    @if not $o-navbar-use-pills {
+        .o_menu_systray_item > .btn {
+            /* This line is extracted from `%-main-navbar-entry-spacing` */
+            line-height: calc(var(--o-navbar-height) - #{$o-navbar-padding-v * 2});
+        }
+    }
+}
```

Comment:
I am still thinking the same thing as earlier: why is this necessary, why is this line-height not inherited instead of duplicated and why is this a necessary website override?

Reply:
Line removed at the end